scripts/check_node_version.js

```diff
@@ -1,13 +0,0 @@
-const semver = require("semver");
```

Comment:
Keep this file, its to prevent carbon contributors using an old version of node. We still want to support node v14 while it's in maintenance lts.

Reply:
Ah my bad I thought we were saying any version is now ok, I'll add it back in 👍

Reply:
any version >= 14 should work.